src/messages/messages.stache

```diff
@@ -1,4 +1,5 @@
-<can-import from="donejs-chat/models/message" />
+<can-import from="~/models/message" />
```

Comment:
The import here and in `src/messages/messages.js` should be kept the same as what’s in the [guide](https://donejs.com/Guide.html).

Reply:
@chasenlehara can't we correct the guide to do ~/ i think this way re usability raises a lot as it don't depends on the package name any more you can copy the folder into a other app and it just works please discusse that for me with the others i will now fallback to donejs-chat but i prefer to make that clear all over since the guides are from a timespan where this was not possible

Reply:
In general, I believe it’s better to show more standard syntax than something tool-specific, so I think showing relative paths is better than the steal-specific `~` syntax.

Reply:
@chasenlehara but this is a donejs project so its donejs specific steal is not a optional tool in the donejs chain or i am wrong?

Reply:
You’re not wrong, and I understand your argument, I just still believe it’s better to show the relative path because `~` requires the user to know what that means and how it resolves. 🙂